Labyrinth/src/GuiTile.java

```diff
@@ -24,5 +24,48 @@ public void paint(Graphics g, Point topLeft, Point bottomRight){
 				width,
 				height
 				);
+		g.setColor(Color.GRAY);
+		//fill center:
+		g.fillRect(
+				topLeft.x + width/3,
+				topLeft.y + height/3,
+				width/3,
+				height/3);
+		//fill up, down, left, right:
+		if(this.tile.unblocked_directions[Tile.direction("Up")]) {
```

Comment:
We could probably abstract this into a method, 
`public void isUnblocked(String direction){`